layers/+irc/erc/packages.el

```diff
@@ -182,6 +182,8 @@
   (spacemacs|define-custom-layout "@ERC"
     :binding "E"
     :body
+    (add-hook 'erc-mode #'(lambda ()
```

Comment:
When there are several expression in a `:property` block, we have for conventions to group them into a `(progn …`.